research-software-directory/RSD-as-a-service
# feat: add community keywords

# Add keywords to community

Closes #1196

Changes proposed in this pull request:
* Community maintainer can add keywords 
* The keywords are shown in the community card in the overview
* The keywords are shown on the community page
* Changed card design: 
    * the number of software items is placed bottom right (similar to software card)
    * the keywords are added to card (similar to software card)

How to test:
* `make start` to build and created test data
* login as rsd admin
* change community settings, add keywords. The keywords should be shown on the page
* conform that keywords are also shown in the card

## Example community overview using keywords
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/9541608f-8f16-4d37-8979-0ce9f227b32a)

## Example community settings with keywords
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/0c419a1a-9cc8-421c-a026-884e3c5e2e78)


PR Checklist:
